### ISSUE TITLE
Comments are not exported when exporting to .odt, .PDF or markdown files

### ISSUE THREAD

**Issue by nekohayo** [ether/ep_comments_page#190]
I suppose figuring out a way to represent comments in markdown could be tricky, but leaving them out when exporting to rich formats like OpenDocument (ODT) and PDF is some pretty serious data loss that can undermine users' trust in the system; they may not notice until it is too late when they reopen the exported document after the etherpad instance is gone/expired...

**Comment by JohnMcLear** [comment]
I agree re odt, we love our libre office brothers and sisters.  But pdf....   Do Adobe even have an open spec for this?! 

Does soffice.bin have support for comments in html to odt?  If so, point me at the docs and I will make sure the html we pass to soffice is compliant.

Fwiw we pass html from etherpad to soffice which is a libre office tool for converting files :)

**Comment by nekohayo** [comment]
I don't know the PDF ISO spec, but I suspect comments and annotations are very standard in there. Otherwise we wouldn't have annotation comments & highlighting in [Evince](https://wiki.gnome.org/Apps/Evince) with the [poppler](https://en.wikipedia.org/wiki/Poppler_(software)) library... I hope someone smarter than me will point out where exactly this is in the spec.

As for LibreOffice being used as a converter and whether this supports comments, maybe @mmeeks would know or would be able to bring this question to the attention of someone who knows?
